privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -261,52 +352,55 @@ def perform_prediction(
         predictions = meta_classifier.predict(x=[feature_extraction_target_model])
         return predictions
 
-    @staticmethod
-    def output_attack(predictions_ratios: Dict[float, np.ndarray]) -> string:
+    def output_attack(self, predictions_ratios) -> Tuple[str, Dict[str, float]]:
         """
         Determination of prediction with highest probability.
```

Comment:
It is more common to start the description of a _method_ with a verb. Because a method is something that does something. Could be "Calculates the prediction..."